<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DLG_SetCanvasSettings</class>
 <widget class="QDialog" name="DLG_SetCanvasSettings">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>150</width>
    <height>141</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <widget class="QLabel" name="lbl_height">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>60</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Height</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_okay">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>110</y>
     <width>60</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Okay</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_width">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Width</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_width">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>61</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>2</number>
   </property>
   <property name="maximum">
    <number>8192</number>
   </property>
   <property name="value">
    <number>200</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_height">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>80</y>
     <width>61</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>2</number>
   </property>
   <property name="maximum">
    <number>8192</number>
   </property>
   <property name="value">
    <number>200</number>
   </property>
  </widget>
  <widget class="QLineEdit" name="textEdit_name">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_name">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Name</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_cancel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>60</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
